pt-BR/mail/chrome/messenger/AccountWizard.dtd

```diff
@@ -23,26 +23,26 @@
 <!ENTITY fullnameLabel.label "Nome:">
 <!ENTITY fullnameLabel.accesskey "N">
 
-<!ENTITY emailLabel.label "Email:">
+<!ENTITY emailLabel.label "E-mail:">
 <!ENTITY emailLabel.accesskey "E">
-<!ENTITY emailDesc.label "Enter your email address; this is the address others will use to send email to you">
+<!ENTITY emailDesc.label "Digite seu endereço de e-mail; este é o endereço que outras pessoas usarão para enviar e-mails a você">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY emailDesc.label "Digite seu endereço de correio eletrônico; este é o endereço que outras pessoas usarão para enviar mensagens a você">